webapp/help_post.py

```diff
@@ -0,0 +1,35 @@
+from sendgrid import Email, sendgrid
```

Comment:
Comment should be here. Something that explains the general purpose of the file :)
"""
Contains functions used for sending the help email ... 
"""